com.Cisco.PacketTracer.yml

```diff
@@ -0,0 +1,44 @@
+app-id: com.Cisco.PacketTracer
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: packettracer
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --persist=pt # App config, logs, templates, saves stored in: ~/pt
+
+modules:
+  - name: packettracer
+    buildsystem: simple
+    build-commands:
+      - mkdir packettracer
+      - ar -x CiscoPacketTracer_821_Ubuntu_64bit.deb --output=packettracer
+      - tar -xvf packettracer/control.tar.xz --directory=packettracer
+      - tar -xvf packettracer/data.tar.xz --directory=packettracer
+      - chmod -R 755 packettracer
+      - mv packettracer/usr/share packettracer/share
+      - rm -d packettracer/usr
+      - cp -rf packettracer/. /app/
+      - sed -i 's/\/opt\/pt/\/app\/opt\/pt/g' /app/opt/pt/packettracer
+      - install -D /app/opt/pt/packettracer /app/bin/packettracer
+      - install -D /app/opt/pt/art/app.png /app/share/icons/hicolor/48x48/apps/$FLATPAK_ID.png
+      - install -D -t /app/share/metainfo $FLATPAK_ID.metainfo.xml
+      - install -D -t /app/share/applications $FLATPAK_ID.desktop
+    sources:
+      - type: file
+        url: https://gitlab.com/ezequielramos/cisco-packet-tracker/-/raw/main/CiscoPacketTracer_821_Ubuntu_64bit.deb
+        # As you can see on the official page, the sha1 being validated is the
+        # official sha1. So you can be sure that this file is the same one that
+        # cisco is providing.
+        # https://www.packettracernetwork.com/download/download-packet-tracer.html
```

Comment:
The official site requires a log in. I'm not sure how legal it is, to use a Package, that you should not get without log in, hosted by a 3rd Party.

Reply:
This is likely not acceptable if the vendor website requires login

Reply:
at best the use of extra-data is required.

Reply:
I tried to contact Cisco about publishing the packet tracer to flathub. I got no response from them. I tried to find something specific about that on their licenses, but I couldn't find anything specific about free software or Packet Tracer, only about paid products or limited/specific devices software.

Even if the user downloads the software from flathub, the software will ask the user to log in using the Cisco account in order to open it. So the account is still needed to use Packet Tracer.

Reply:
this is not a reason good enough to clain redistribution is allowed. If the vendor put the download under a login, then it's likely that the vendor want to restrict redistribution. Unless stated otherwise.